### DISCUSSION TITLE
Configure Display name

### DISCUSSION THREAD

**Discussion by crjo** [nordeck/jitsi-keycloak-adapter#14] [Q&A] [answered]
Hi,

The adapter works fine but the display name is the user mail address as :
![image](https://github.com/nordeck/jitsi-keycloak-adapter/assets/5144838/373457e8-f469-4192-914c-0f4edddfc799)

Is there a way to have the display name set as the full name like : 
![image](https://github.com/nordeck/jitsi-keycloak-adapter/assets/5144838/a73cc03e-ad9a-4419-973e-9009eb02fc35)
or
![image](https://github.com/nordeck/jitsi-keycloak-adapter/assets/5144838/542eefa8-a954-495c-9950-969dfdc5ef48)

Thanks

**Comment by emrahcom** [comment, marked as answer]
Hi @crjo,

This value comes from the `name` field of the token payload generated by `jitsi-keycloak-adapter`. I keep the related codes in a seperate file to simplify customizations. The related file is [context.ts](https://github.com/nordeck/jitsi-keycloak-adapter/blob/main/context.ts)

It uses `preferred_username` of `Keycloak` by default but you can update these codes to set another value for `name`.

**Reply by crjo** [reply]
Thanks it's an easy move!
